src/styles/variables.scss

```diff
@@ -4,14 +4,15 @@ $brand-success: #5cb85c;
 $brand-warning: #f0ad4e;
 $brand-danger: #d9534f;
 $brand-info: #5bc0de;
-
+$border-color: #EFEFEF;
 $text-color: #444;
 
 $font-size-base: 14px;
 $font-family-sans-serif: 'Montserrat', sans-serif;
+$font-family-regular: 'Montserrat-regular', sans-serif;
 
 $navbar-inverse-bg: transparent;
 $navbar-inverse-color: #fff;
 $navbar-inverse-link-color: #fff;
-
 @import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables";
```

Comment:
That was my addition. Cause I want to use a bootstrap variable within our variables.